parking_permits/tests/test_views.py

```diff
@@ -60,6 +62,19 @@ def get_validate_order_data(talpa_order_id, talpa_order_item_id):
     }
 
 
+def get_validate_subscription_data(
```

Comment:
The wording of this function is not quite clear. Is the subscription data assumed to be already validated, does the function do any validation, or does this provide data for validation?

Reply:
Fix typo / word `validate` to `validated`.